### PR TITLE
Aded BUSCO ideogram for ``GENOME`` mode

### LINKED ISSUE #98: `BUSCO_IDEOGRAM` for `GENOME` mode
So far, the module `BUSCO_IDEOGRAM` works only on `GENOME_AND_ANNOTATION`. We should make it also functional for the `GENOME` mode.

Just to point it out, the `BUSCO_IDEOGRAM` on `GENOME` mode won't work on the test datasets, see issue #97 for more information.

### PR DESCRIPTION
**Pull request by FernandoDuarteF**
Close #98.

## Changes

1. Added `plot_makers1.R` and adapted `genome_ideogram.nf` from the BUSCO [support protocol 2](https://gitlab.com/ezlab/busco_protocol/-/tree/main/support_protocol2/).
2. Changed version of BUSCO in `busco/busco/main.nf` module to v5.8.0 (see #97).

## Comments

Probably some code cleaning is necessary before merging, specially regarding the **R scripts**. Other than that, this is working as expected, at least on the test dataset.

### CODE REVIEW

**Comment by FernandoDuarteF** [comment]
Perhaps it would be a good idea to have a single module for the BUSCO ideogram, as now there is one for genome only mode (renamed to `genome_only_busco_ideogram.nf`) and one for genome and annotation (renamed to `genome_annotation_busco_ideogram.nf`).

We shoould also rename some scripts in the `/bin` folder to make them more descriptive. I'll deal with this later.

**Comment by chriswyatt1** [comment]
We could just add this to another issue for now. Yes, would make sense to have in once module, but I think this way is fine for now

File: modules/local/genome_only_busco_ideogram.nf
Comment (chriswyatt1) [inline modules/local/genome_only_busco_ideogram.nf:1]: We should not have this: `$projectDir/bin/plot_busco_ideogram.R`

Just make sure the shebang is correct in the R code

File: subworkflows/local/genome_and_annotation.nf
Comment (chriswyatt1) [inline subworkflows/local/genome_and_annotation.nf:1]: Remove all the blanked out comments that we don't need:

`//removed this temporarily:, ch_karyotype`

**Review by chriswyatt1 — CHANGES_REQUESTED** [review]
Just a few little things to change, commented later

File: modules/local/genome_only_busco_ideogram.nf
Comment (FernandoDuarteF) [reply modules/local/genome_only_busco_ideogram.nf:1]: Yep, I forgot to change this

**Comment by FernandoDuarteF** [comment]
I thinks this is ready to merge.

**Pull request merged**